### DISCUSSION TITLE
Unknown dynamic nested routes

### DISCUSSION THREAD

**Discussion by Der-Alex** [nuxt/nuxt#17982] [Questions] [answered]
Hi. In V2 there was the option to create a _.vue for unknown dynamic nested routes. https://nuxtjs.org/docs/features/file-system-routing/#unknown-dynamic-nested-routes
What is the equvalent in Nuxt 3? I can't find a solution in the documentation. 

Kind Regards :)

**Comment by viii47** [comment]
See https://v3.nuxtjs.org/docs/directory-structure/pages#dynamic-routes

The new dynamic routing template is `[dynamic.vue]` which then can be accessed with `route.params.dynamic`

**Reply by Der-Alex** [reply]
Thanks for your reply @viii47 but this does not fit into my problem. 
Nuxt 2 had the possibility to create an unknown deep nested url route like `localhost:3000/foo/bar/crux` when using _.vue. 
When using [whatever].vue you only have one level of nesting like `localhost:3000/foo` but I need the deep nesting. Therefore I have to create a nesting folder structure what I dont want. in Vue 2 the _.vue file was enough.

I hope that makes it more clear what I mean.

**Comment by Der-Alex** [comment, marked as answer]
OK I figured it out by myself. It is the [...slug].vue example in the nuxt 3 docs. So when you name the file like that, you have deep nestable catch-all route.
